internal/server/server.go

```diff
@@ -269,15 +269,31 @@ func (s *AsyncServer) handleClientEvent(event iomultiplexer.Event) error {
 }
 
 func (s *AsyncServer) executeCommandToBuffer(redisCmd *cmd.RedisCmd, buf *bytes.Buffer, c *comm.Client) {
-	s.shardManager.GetShard(0).ReqChan <- &ops.StoreOp{
-		Cmd:      redisCmd,
-		WorkerID: "server",
-		ShardID:  0,
-		Client:   c,
+	// breaking down single command to multiple command
+	// Len(cmdsBkp) also let worker know about number of shards to wait
+	cmdsBkp := []cmd.RedisCmd{}
+
+	val, ok := WorkerCmdsMeta[redisCmd.Cmd]
+	if !ok {
+		cmdsBkp = append(cmdsBkp, *redisCmd)
+	} else {
+		switch val.CmdType {
+		case Global:
+			buf.Write(val.RespNoShards(redisCmd.Args))
+			return
+
+		case SingleShard, Custom:
```

Comment:
It appears we haven't added any Custom commands. We can remove it for now.

Reply:
Again for future reference just to make sure we will be working with the 4 categories always, I have kept this.